Update Contributing Instructions (#72)
InnerSourceCommons/ispo-working-group: CONTRIBUTING.md

    WG[ISPO Working Group] --> KB[Kanban Board]
    WG --> GD[Reviews]
    WG --> MU[Meetings]
```

1. Attend a working group meeting.

    * The [InnerSource Commons Slack] [#ispo-working-group] community will post the next meeting.

2. Review others' content.

    * Find something interesting on the [Kanban board].
    * Write a comment on the card indicating your interest.

3. [Kanban board].

InnerSource solutions to challenges are discovered, produced, and spread in this working group.

> Contributions usually become added or improved content in the [Managing InnerSource Projects] book, [InnerSource Patterns] book, or [InnerSource Learning Path] segment.

Reviewer: ```suggestion
Contributions are usually added to or improve content in the [Managing InnerSource Projects] book, [InnerSource Patterns] book, or [InnerSource Learning Path] segment.
```

Slides, spreadsheets, and other artifacts not easily represented elsewhere can go in the [ISPO Working Group GDrive].
Be sure to attribute all the contributors to the resulting work (e.g., in the _Authors_ or _Acknowledgements_ sections of Patterns).

> To contribute, get access to contribute to the board by [joining the ISPO Working Group team].

Reviewer: ```suggestion
To contribute, get access to the board by [joining the ISPO Working Group team].
```

A regular meeting proposes new challenges, discusses ideas, and demos solutions.

  1. Create a new card in the **No Status** column describing your contribution and the challenge it helps solve.
  1. Assign the new card to yourself by clicking the link on the card.
  1. Check if your contribution relates to existing [InnerSource Patterns].

>   1. Learn more about the next Working Group meeting by joining [#ispo-working-group] in [InnerSource Commons Slack].

Reviewer: ```suggestion
  1. Learn more about the next Working Group meeting by joining [#ispo-working-group] in the [InnerSource Commons Slack] workspace.
```
